thangs_importer.py

```diff
@@ -0,0 +1,303 @@
+import webbrowser
+import urllib.parse
+import os
+import time
+import math
+from urllib.request import urlopen
+import tempfile
+import urllib.request
+import requests
+import os
+import importlib
+import threading
+import asyncio
+# from . import addon_updater_ops
+import socket
+import json
+import shutil
+from .config import ThangsConfig, get_config
+from uuid import UUID
+
+import bpy
+from bpy.types import WindowManager
+import bpy.utils.previews
+from bpy.types import (Panel,
+                       PropertyGroup,
+                       Operator,
+                       )
+from bpy.props import (StringProperty,
+                       PointerProperty,
+                       FloatVectorProperty,
+                       )
+from bpy_extras.object_utils import AddObjectHelper, object_data_add
+from bpy.app.handlers import persistent
+
+_thangs_api = None
+
+def get_thangs_api():
+    global _thangs_api
+    return _thangs_api
+
+
+def initialize_thangsAPI(callback):
```

Comment:
nit: python conventions are usually that methods are all `snake_case` and not capitalized.  So, let's make this `initialize_thangs_api`